06-Plotting-Time-Series-Rasters-in-R.Rmd

```diff
@@ -229,15 +241,34 @@ we adjust the layout to be a matrix of 4 columns and 4 rows.
 ```{r adjust-layout }
 #use level plot to create a nice plot with one legend and a 4x4 layout.
 levelplot(NDVI_stack,
-          layout=c(5, 3), #create a 4x4 layout for the data
+          layout=c(5, 3), #create a 5x3 layout for the data
           col.regions=cols, #add a color ramp
-          main="Landsat NDVI - Julian Days \nHarvard Forest 2014",
+          main="Landsat NDVI - Julian Days \nHarvard Forest 2011",
           names.attr=rasterNames)
 ```
 
-CHALLENGE -- change the X in each layer name to `Julian_` to label each plot.
-{: .notice }
+#Challenge: Divergent Color Ramps 
+When we used `gsub` to modify the tile labels we just used Day.  However, it
+would be nice to be more descriptive and use the full "JulianDay_" so that it is
+clear what is being denoted.  
 
+Create a plot with this change and use a divergent brown to green color ramp to
```

Comment:
Did we explain what divergent vs sequential color ramp is? this might confuse people. 
i went back and forth with color brewer. it's nice but it has limitations. And it's own set of terms.  if we use this language we might want to explain it a bit above. or we could explain it in a challenge and let people experiment more with colors.